src/engraving/dom/scorefile.cpp

```diff
@@ -129,6 +130,26 @@ bool Score::loadStyle(const String& fn, bool ign, const bool overlap)
     return false;
 }
 
+bool Score::loadStyle(muse::ByteArray&& data, bool ign, bool overlap)
+{
+    TRACEFUNC;
+
+    muse::io::Buffer buffer(std::move(data));
+
+    if (buffer.open(IODevice::ReadOnly)) {
```

Comment:
It's better not to duplicate the method, but to add another one that take IODevice

Reply:
added, thanks!